src/components/LastFlowRun.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <FlowRunName :flow-run="flowRun" />
+</template>
+
+<script lang="ts" setup>
+  import { computed } from 'vue'
+  import { FlowRunName } from '@/components'
+  import { useLastFlowRun } from '@/compositions'
+  import { UnionFilter } from '@/models'
+
+  const props = defineProps<{
+    flowId?: string,
+  }>()
+
+  const filter = computed(() => ({
+    flows: {
+      id: [props.flowId],
+    },
+  })) as UnionFilter
+
+  const { flowRun } = useLastFlowRun(filter)
```

Comment:
Its preferred to just use the getter directly for this sort of thing rather than using a computed. 

```suggestion
  const { flowRun } = useLastFlowRun(() => ({
    flows: {
      id: [props.flowId],
    },
  }))
```